lib/provisioner/worker/plugins/providers/fog_provider/lib/fog_provider/joyent.rb

```diff
@@ -158,23 +164,27 @@ def confirm(inputmap)
         #   centos: vdb1 already mounted at /data
         if vdb1
           # TODO: check that vdb1 is mounted at /data, for now assume it is
-          log.debug 'Assuming /dev/vdb1 is mounted at /data, if this is not the case, file an issue'
+          log.debug 'Assuming /dev/vdb1 is mounted at /data or at /mnt, if this is not the case, file an issue'
         elsif vdb
           begin
-            mounted = false
-            ssh_exec!(ssh, 'if grep "vdb /data " /proc/mounts ; then /bin/false ; fi', 'Checking if /dev/vdb mounted already')
+            data_mounted = false
+            ssh_exec!(ssh, 'if grep "vdb /data " /proc/mounts ; then echo "/dev/vdb is mounted"; else /bin/false ; fi', 'Checking in /proc/mounts whether /dev/vdb mounted already')
+            data_mounted = true
           rescue
-            mounted = true
+            log.debug 'Disk /dev/vdb is not mounted to /data'
+            log.debug "data_mounted = #{data_mounted}"
```

Comment:
+1

###
Random auto-corrects and typos are my special gift to you. When I forward they are from others. 

> On Mar 4, 2015, at 10:33 PM, dereklwood <notifications@github.com> wrote:
> 
> In lib/provisioner/worker/plugins/providers/fog_provider/lib/fog_provider/joyent.rb:
> 
> >            rescue
> > -            mounted = true
> > +            log.debug 'Disk /dev/vdb is not mounted to /data'
> > +            log.debug "data_mounted = #{data_mounted}"
> remove this log line... not needed
> 
> —
> Reply to this email directly or view it on GitHub.
>